src/view/placeholder.js

```diff
@@ -44,12 +46,16 @@ export function attachPlaceholder( view, element, placeholderText, checkFunction
 }
 
 /**
- * Removes placeholder functionality from given element.
+ * Removes the placeholder functionality from a given element.
  *
  * @param {module:engine/view/view~View} view
  * @param {module:engine/view/element~Element} element
  */
-export function detachPlaceholder( view, element ) {
+export function removePlaceholder( view, element ) {
```

Comment:
I was looking for usage of this function but I found none (only tests). If it is not needed maybe the change in this function is not needed too.

Reply:
AFAIR it is used in Letters, isn't it?